ui/lib/core/addon/templates/components/alert-inline.hbs

```diff
@@ -1,8 +1,18 @@
-<Icon @name={{this.alertType.glyph}} class={{this.alertType.glyphClass}} />
-<p class={{this.textClass}} data-test-inline-error-message>
-  {{#if (has-block)}}
-    {{yield}}
+<div
+  {{did-update this.refresh @message}}
```

Comment:
I like how you got around this requirement. This is basically to force some kind of more dramatic view change if the message itself changes?

Reply:
Yeah! Before updating the count, the loading spinner displays briefly
![fake-load](https://user-images.githubusercontent.com/68122737/170316686-d11754a5-2dd0-4b0f-ae43-a5fdc4a03852.gif)